src/tests/unit/tests/DetailsView/details-view-container.test.tsx

```diff
@@ -271,6 +271,7 @@ describe('DetailsViewContainer', () => {
                 issueTrackerPath={issueTrackerPath}
                 rightPanelConfiguration={rightPanelConfiguration}
                 switcherNavConfiguration={switcherNavConfiguration}
+                userConfigurationStoreData={storeMocks.userConfigurationStoreData}
```

Comment:
I'd seen this change on a lot of files... any thoughts on how to avoid this multiple-files change when we need to add store data to the props of a leaf component?

Reply:
Yeah, this is painful for me, but currently I cannot think of a solution other than use Context.